src/allotropy/parsers/moldev_softmax_pro/softmax_pro_parser.py

```diff
@@ -97,7 +97,11 @@ def _get_model(self, file_name: str, data: Data) -> Model:
                     for position in plate_block.iter_wells()
                 ],
                 calculated_data_aggregate_document=CalculatedDataAggregateDocument(
-                    calculated_data_document=self._get_calc_docs(data)
+                    calculated_data_document=(
+                        None
```

Comment:
What you can do here instead is check if `_get_calc_docs` returns any items and based on that include or not the `calculated_data_aggregate_document`, which is optional, to avoid having an "aggregate doc" without any "document".